html/webappapis/dynamic-markup-insertion/opening-the-input-stream/bailout-exception-vs-return-origin.sub.window.js

```diff
@@ -0,0 +1,82 @@
+document.domain = "{{host}}";
+
+async_test(t => {
+  const iframe = document.body.appendChild(document.createElement("iframe"));
+  t.add_cleanup(() => { iframe.remove(); });
+  iframe.onload = t.step_func(() => {
+    // Here, the entry settings object is still the iframe's. Delay it in such
+    // a way that makes the entry settings object the top-level page's, but
+    // without delaying too much that the parser becomes inactive. A microtask
+    // is perfect as it's executed in "clean up after running script".
```

Comment:
In addition to the factored-out common comment, it's good to explain in each case when "clean up after running script" occurs. Here, it occurs in the `</script>` tag in the subframe.